app/controllers/flood_risk_engine/enrollments/steps_controller.rb

```diff
@@ -32,6 +36,46 @@ def update
         end
       end
 
+      def process_postcode # rubocop:disable Metrics/AbcSize
+        # We redirect here from update when dealing with page containing Postcode entry and form validated
+        enrollment.go_forward
+        enrollment.save
+
+        # form & step cache results so don't return what we need after go_forward - so this ensures the right values
+        address_form = Steps::FormObjectFactory.form_object_for(enrollment.step, enrollment)
+
```

Comment:
If you break out address_form to a method, you can remove the # rubocop:disable Metrics/AbcSize
```
def address_form
  @address_form ||= Steps::FormObjectFactory.form_object_for(enrollment.step, enrollment)
end
```